README.md

```diff
@@ -1,11 +1,23 @@
-AdvancedTabOverlay is a plugin for Spigot that has not been documented by its author yet.
+[Dev Builds]: https://ci.codecrafter47.de/job/AdvancedTabOverlay/
+[ProtocolLib]: https://www.spigotmc.org/resources/1997/
+[Wiki]: https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki
+[spigot]: https://www.spigotmc.org/resources/advancedtaboverlay.78905/
 
-File locations
---------------
-- `src/site/markdown`: Markdown files that will be included in the generated site
-- `src/main/java`: Main plugin source files
-- `src/test/java`: Test source files
+# AdvancedTabOverlay
+AdvancedTabOverlay is a tab list plugin for Spigot, which provides highly customisable features including but not limited to:
+- Custom slots with icon, text and ping
+  - Those can even be animated!
+- Displaying of different tab lists depending on various conditions
+  - Per Player
+  - Per World
+  - Per Group
+  - ...
+- Supports multiple plugins including LuckPerms, Vault, EssentialsX and more.
+- Tons of placeholders to use within your tab lists.
```

Comment:
We could mention PlaceholderAPI support here?

Reply:
Added it